PR template (#1139)
vector-im/element-x-android: .github/pull_request_template.md

## Screenshots / GIFs

<!--

> We have screenshot test in the project, so adding screenshots is not mandatory, as far as there

Reviewer: ```suggestion
We have screenshot tests in the project, so attaching screenshots to a PR is not mandatory, as far as there
```

is a Composable Preview covering the changes. In this case, the change will appear in the file diff.

> Note that all the UI composable should be covered by a Composable Preview.

Reviewer: ```suggestion
Note that all the UI composables should be covered by a Composable Preview.
```

<!-- Depending on the Pull Request content, it can be acceptable if some of the following checkboxes stay unchecked. -->

> - [ ] Changes has been tested on an Android device or Android emulator with API 23

Reviewer: ```suggestion
- [ ] Changes have been tested on an Android device or Android emulator with API 23
```
